src/linter/report.go

```diff
@@ -455,6 +455,20 @@ function performance_test() {}`,
 			Default: true,
 			Comment: `Report linter error.`,
 		},
+
+		{
+			Name:    "magicMethod",
+			Default: true,
+			Comment: `Report error in magic methods.`,
+			Before: `class Foo {
+  private function __call($method, $args) {} // The magic method __call() must have public visibility
+  static function __set($name, $value) {} // The magic method __set() cannot be static
+}`,
+			After: `class Foo {
+  public function __call($method, $args) {} // Ok
+  function __set($name, $value) {} // Ok
```

Comment:
It's generally a bad practice to omit access modifiers in classes. "public" is implied, but our examples shouldn't display a code that smells. :)